web/components/admin/ApplicantInfo/index.js

```diff
@@ -36,13 +38,34 @@ class ApplicantInfo extends React.Component {
 
   render() {
     const { className } = this.props;
+    const shortInfo = this.getApplicantShortInfo();
```

Comment:
Sorry I failed to mention this earlier but for the applicant assessment page we only want to view a handful of fields.
https://www.notion.so/nwplus/nwHacks-admin-meeting-43e93fb8ed8b453c92a1ab1b23e6f9be#aed998e0452647368ce744cf02d60731

The decision was to exclude fields that might introduce bias in the grading process and leave only those that are required for grading an applicant. You can see these fields on the mocks, but for your convenience they are: first hackathon, github, personal site, linkedin, resume, and long answers.

However, as mentioned in the notion doc above the plan is to have a separate page (an "Applicants" page) the will look identical to the "Assessment" page (without the score buttons) where you can view all the information about an applicant. So instead of deleting the extra fields that you have right now, maybe hide them conditionally according to some sort of page type prop.